specification/trace/semantic_conventions/messaging.md

```diff
@@ -217,6 +217,23 @@ For Apache Kafka producers, [`peer.service`](./span-general.md#general-remote-se
 The `service.name` of a Consumer's Resource SHOULD match the `peer.service` of the Producer, when the message is directly passed to another service.
 If an intermediary broker is present, `service.name` and `peer.service` will not be the same.
 
+#### CloudEvents
+
+For CloudEvents clients, the following attributes are defined:
+
+<!-- semconv messaging.cloudevents -->
+| Attribute  | Type | Description  | Examples  | Required |
+|---|---|---|---|---|
+| `messaging.cloudevents.id` | string | The [id](https://github.com/cloudevents/spec/blob/master/spec.md#id) identifies the event within the scope of a producer. | `123e4567-e89b-12d3-a456-426614174000`; `producer-1` | No |
```

Comment:
ID is pretty generic, this could be about the event, consumer, producer, topic, .... How about `event_id`?

Reply:
I wanted to keep as is from cloudevents but I think it makes sense to change it. Done.